### ISSUE TITLE
[a11y] Cannot open Variants on Agent view

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1410]
database: `saiab`

**Steps:**
1. Open an agent QCBX
2. Add a varient
3. See error

![image](https://user-images.githubusercontent.com/37256050/167187472-41df8b19-1b5f-4cab-820f-cd36ed4dd459.png)

**Comment by grantfitzsimmons** [comment]
![image](https://user-images.githubusercontent.com/37256050/167204865-10457b3e-6eff-4518-bba5-70e3bb74f6df.png)

Same error occurs when opening an agent record from a QCBX in Exchange In

**Comment by maxxxxxdlp** [comment]
Duplicate of #1126
